### PR TITLE
Run Busola UI test on post-main and rename to Integration test

### PR DESCRIPTION
**Pull request by Sawthis**
**Description**

Changes proposed in this pull request:

- Run Busola UI tests on post-main
- Rename UI to Integration test

**Related issue(s)**

### CODE REVIEW

**Review by parostatkiem — DISMISSED** [review]

File: prow/scripts/provision-vm-and-start-busola-k3s.sh [hunk 1]
```diff
@@ -51,9 +51,9 @@ RANDOM_ID=$(openssl rand -hex 4)
 
 LABELS=""
 if [[ -z "${PULL_NUMBER}" ]]; then
-    LABELS=(--labels "branch=$PULL_BASE_REF,job-name=busola-ui-test")
+    LABELS=(--labels "branch=$PULL_BASE_REF,job-name=busola-integration-test")
```
Comment (dekiel) [inline prow/scripts/provision-vm-and-start-busola-k3s.sh:54]:
```suggestion
    LABELS=(--labels "branch=$PULL_BASE_REF,job-name=busola-integration-test-k3s")
```

File: development/tools/jobs/kyma/kyma_integration_test.go [hunk 2]
```diff
@@ -314,7 +314,7 @@ func TestKymaIntegrationJobPeriodics(t *testing.T) {
 	tester.AssertThatHasExtraRepoRefCustom(t, vmsCleanerPeriodic.JobBase.UtilityConfig, []string{"test-infra"}, []string{"main"})
 	assert.Equal(t, tester.ImageProwToolsCurrent, vmsCleanerPeriodic.Spec.Containers[0].Image)
 	assert.Equal(t, []string{"bash"}, vmsCleanerPeriodic.Spec.Containers[0].Command)
-	assert.Equal(t, []string{"-c", "/prow-tools/vmscollector -project=${CLOUDSDK_CORE_PROJECT} -vmNameRegexp='.*-integration-test-.*|busola-ui-test-.*' -jobLabelRegexp='.*-integration$|busola-ui-test' -dryRun=false"}, vmsCleanerPeriodic.Spec.Containers[0].Args)
+	assert.Equal(t, []string{"-c", "/prow-tools/vmscollector -project=${CLOUDSDK_CORE_PROJECT} -vmNameRegexp='.*-integration-test-.*|busola-integration-test-.*' -jobLabelRegexp='.*-integration$|busola-integration-test' -dryRun=false"}, vmsCleanerPeriodic.Spec.Containers[0].Args)
```
Comment (dekiel) [inline development/tools/jobs/kyma/kyma_integration_test.go:317]:
```suggestion
	assert.Equal(t, []string{"-c", "/prow-tools/vmscollector -project=${CLOUDSDK_CORE_PROJECT} -vmNameRegexp='.*-integration-test-.*|busola-integration-test-.*' -jobLabelRegexp='.*-integration$|busola-integration-test-k3s' -dryRun=false"}, vmsCleanerPeriodic.Spec.Containers[0].Args)
```

File: prow/scripts/provision-vm-and-start-busola-k3s.sh [hunk 3]
```diff
@@ -51,9 +51,9 @@ RANDOM_ID=$(openssl rand -hex 4)
...
 else
-    LABELS=(--labels "pull-number=$PULL_NUMBER,job-name=busola-ui-test")
+    LABELS=(--labels "pull-number=$PULL_NUMBER,job-name=busola-integration-test")
```
Comment (dekiel) [inline prow/scripts/provision-vm-and-start-busola-k3s.sh:56]:
```suggestion
    LABELS=(--labels "pull-number=$PULL_NUMBER,job-name=busola-integration-test-k3s")
```

File: templates/data/kyma-integration-data.yaml [hunk 4]
```diff
@@ -529,7 +529,7 @@ templates:
                   command: "bash"
                   args:
                     - "-c"
-                    - "/prow-tools/vmscollector -project=${CLOUDSDK_CORE_PROJECT} -vmNameRegexp='.*-integration-test-.*|busola-ui-test-.*' -jobLabelRegexp='.*-integration$|busola-ui-test' -dryRun=false"
+                    - "/prow-tools/vmscollector -project=${CLOUDSDK_CORE_PROJECT} -vmNameRegexp='.*-integration-test-.*|busola-integration-test-.*' -jobLabelRegexp='.*-integration$|busola-integration-test' -dryRun=false"
```
Comment (dekiel) [inline templates/data/kyma-integration-data.yaml:532]:
```suggestion
                    - "/prow-tools/vmscollector -project=${CLOUDSDK_CORE_PROJECT} -vmNameRegexp='.*-integration-test-.*|busola-integration-test-.*' -jobLabelRegexp='.*-integration$|busola-integration-test-k3s' -dryRun=false"
```

**Review by dekiel — CHANGES_REQUESTED** [review]
Please check suggestions.

**Review by dekiel — APPROVED** [review]

**Pull request merged**